src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  transition-duration: 1s;
+
+  &__image {
+    box-sizing: border-box;
+    height: 134px;
+    width: 160px;
+    margin-top: 32px;
+    margin-inline: 3px;
+  }
+
+  &__title {
+    transition-duration: 1s;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
```

Comment:
It's not necessary to use 'display: flex;' and 'flex-direction: row;' on the '.card__reviews' class. By default, block-level elements like div (which is probably what you're using for your reviews) display their children in a row. You can remove these properties unless you have a specific reason to use them.